src/Form/matching-configuration.js

```diff
@@ -10,11 +10,17 @@ const matchingConfiguration = {
     /** @type {MatcherConfiguration} */
     matchers: {
         fields: {
-            email: {
-                type: 'email',
+            unknown: {
+                type: 'unknown',
                 strategies: {
-                    cssSelector: 'email',
-                    ddgMatcher: 'email',
+                    ddgMatcher: 'unknown'
+                }
+            },
+            emailAddress: {
+                type: 'emailAddress',
+                strategies: {
+                    cssSelector: 'emailAddress',
+                    ddgMatcher: 'emailAddress',
```

Comment:
This is simpler to read without whitespace, but all I'm doing is:

- Introduce an `unknown` matcher to ensure we return early when certain signals are obvious (like captchas). This has a minor performance impact, but most importantly avoids false-matching with the vendor strategy when we already know it should be `unknwon` (vendor strategies don't have the concept of `forceUnknown`).
- Rename `email` to `emailAddress` because that's how we call it in the rest of the codebase, so this inconsistency just added confusion. If we ever want to go in and simplify all this stuff to only have one name, this brings us closer.